docs/html/cli/pip_install.rst

```diff
@@ -379,7 +379,7 @@ Finding Packages
 
 pip searches for packages on `PyPI`_ using the
 `HTTP simple interface <https://pypi.org/simple/>`_,
-which is documented `here <https://setuptools.readthedocs.io/en/latest/easy_install.html#package-index-api>`_
+which is documented `here <https://setuptools.readthedocs.io/en/latest/deprecated/easy_install.html#package-index-api>`_
 and `there <https://www.python.org/dev/peps/pep-0503/>`_.
```

Comment:
We should remove the setuptools link altogether and link to https://packaging.python.org/specifications/simple-repository-api/ instead.